setup.py

```diff
@@ -163,8 +170,26 @@ def runtest(self):
     keywords="",
     license="BSD",
     long_description="",
-    name="cellprofiler",
-    package_data = {
+    name="CellProfiler",
+    options={
+        "py2app": {
+            "packages": [
+                "cellprofiler",
```

Comment:
So for py2exe, I need to use setuptools.find_packages to find py2exe's packages because py2exe did not find the module .py files and other. I am jealous.

Nothing wrong with this, just a comment why mine is different.